plugins/chat/assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -13,6 +14,7 @@ replaceIcon("d-chat", "comment");
 
 export default {
   name: "chat-setup",
+  before: "hashtag-css-generator",
```

Comment:
This ordering is important, we must have the user's channels loaded into the manager and the hashtag type for the channel registered before proceeding with the hashtag CSS generation.